custom-metrics-stackdriver-adapter/deploy/production/adapter.yaml

```diff
@@ -78,6 +78,26 @@ spec:
         command:
         - /adapter
         - --use-new-resource-model=false
+        - --secure-port=6443
+        ports:
+        - containerPort: 6443
+          name: https
```

Comment:
is this port only for serving the readiness and liveness checks?  if so consider giving it a more descriptive name or just use port number on line 89 and 97 without a name.
If it is also serving the metrics API, then I guess we also need to update the `targetPort: 443` for the Service?

Reply:
Good point! Nice catch!
Updated to `targetPort: 6443` and did an E2E test following https://cloud.google.com/stackdriver/docs/managed-prometheus/hpa#stackdriver-adapter